components/CMakeLists.txt

```diff
@@ -236,6 +236,23 @@ if (UNIX AND NOT APPLE)
 target_link_libraries(components ${CMAKE_THREAD_LIBS_INIT})
 endif()
 
+add_library(osgdb_serializers_nifosg SHARED
+    serializers/nifloader serializers/controller
+    serializers/particle serializers/userdata
+    )
+
+target_link_libraries(osgdb_serializers_nifosg components)
+
+add_library(osgdb_serializers_sceneutil SHARED
```

Comment:
It appears that OSG automatically loads shared libraries based on the namespace of the objects in the tree.  Since some are in NifOsg and others are SceneUtil, we ended up with two shared libraries (icky!).   Perhaps the solution is to move everything into one name space, or even an "OpenMW" namespace so that the serializer would load allo of our serializers at once?  I dunno.   Your guidance would be appreciated.